### ISSUE TITLE
XEP-0227: Portable Import/Export Format for XMPP-IM Servers

### ISSUE THREAD

**Issue by Neustradamus** [tigase/tigase-server#114]
Can you add Portable Import/Export Format for XMPP-IM Servers support?
- https://xmpp.org/extensions/xep-0227.html

**Comment by Neustradamus** [comment]
@arturhefczyc: What is the status of this XEP support?

**Comment by arturhefczyc** [comment]
From what I remember we have/had import implemented pretty well. However, nobody seems to implement export, so really there is no much use for it anyway.
TBH I am not sure if we have the export implemented.

Maybe @woj-tek knows more about it.

**Comment by arturhefczyc** [comment]
By the way, why are you asking about it?

**Comment by arturhefczyc** [comment]
Also, please submit all new issues onto https://tigase.dev as new issues and updates on GitHub are not transferred over to the new system.

**Comment by Neustradamus** [comment]
It is important to be compatible, it is interoperability between XMPP Server for import and export...

**Comment by arturhefczyc** [comment]
I agree, generally. However, certain features require significant effort to implement and maintain them in a working condition. Sometimes, the effort and cost associated with the work cannot be justified.
It seems like this is the case with import/export functionality.
